### PR TITLE
report log change

### PR DESCRIPTION
**Pull request by khushbu-maurya**

### CODE REVIEW

**Comment by admin-vizcentric** [comment]
There were no errors, go have a cup of coffee...

File: src/main/proto/Report.proto [hunk 1]
```diff
@@ -80,6 +80,7 @@ message ReportLog {
     bool enableTicketCreation = 13;
     string viewTicket = 14;
     Query query = 15;
+    string visualizationid = 16;
```
Comment (johibkhan2) [inline src/main/proto/Report.proto]: rename visualizationid to visualizationId

**Comment by admin-vizcentric** [comment]
There were no errors, go have a cup of coffee...

**Review by johibkhan2 — APPROVED** [review]

**Review by admin-vizcentric — APPROVED** [review]

**Comment by admin-vizcentric** [comment]
There were no errors, go have a cup of coffee...

**Pull request merged**